pennylane/queuing.py

```diff
@@ -580,4 +578,4 @@ def process_queue(queue: AnnotatedQueue):
                 )
```

Comment:
the docstring should be updated here, effectively just doing the opposite of [this commit](https://github.com/PennyLaneAI/pennylane/commit/d4a500ae99070eed76658586372b7e1bf1dfc55b)